code/game/objects/items/storage/uplink_kits.dm

```diff
@@ -1,397 +1,397 @@
-	new /obj/item/clothing/mask/chameleon(src)
-	new /obj/item/storage/backpack/chameleon(src)
-	new /obj/item/radio/headset/chameleon(src)
-	new /obj/item/stamp/chameleon(src)
-	new /obj/item/pda/chameleon(src)
-
-//5*(2*4) = 5*8 = 45, 45 damage if you hit one person with all 5 stars.
-//Not counting the damage it will do while embedded (2*4 = 8, at 15% chance)
-/obj/item/storage/box/syndie_kit/throwing_weapons/PopulateContents()
-	for(var/i in 1 to 5)
-		new /obj/item/throwing_star(src)
-	for(var/i in 1 to 2)
-		new /obj/item/paperplane/syndicate(src)
-	new /obj/item/restraints/legcuffs/bola/tactical(src)
-	new /obj/item/restraints/legcuffs/bola/tactical(src)
-
-/obj/item/storage/box/syndie_kit/cutouts/PopulateContents()
-	for(var/i in 1 to 3)
-		new/obj/item/cardboard_cutout/adaptive(src)
-	new/obj/item/toy/crayon/rainbow(src)
-
-/obj/item/storage/box/syndie_kit/romerol/PopulateContents()
-	new /obj/item/reagent_containers/glass/bottle/romerol(src)
-	new /obj/item/reagent_containers/syringe(src)
-	new /obj/item/reagent_containers/dropper(src)
-
-/obj/item/storage/box/syndie_kit/ez_clean/PopulateContents()
-	for(var/i in 1 to 3)
-		new/obj/item/grenade/chem_grenade/ez_clean(src)
-
-/obj/item/storage/box/hug/reverse_revolver/PopulateContents()
-	new /obj/item/gun/ballistic/revolver/reverse(src)
-
-/obj/item/storage/box/syndie_kit/mimery/PopulateContents()
-	new /obj/item/book/granter/spell/mimery_blockade(src)
-	new /obj/item/book/granter/spell/mimery_guns(src)
-
-/obj/item/storage/box/syndie_kit/centcom_costume/PopulateContents()
-	new /obj/item/clothing/under/rank/centcom_officer(src)
-	new /obj/item/clothing/shoes/sneakers/black(src)
-	new /obj/item/clothing/gloves/color/black(src)
-	new /obj/item/radio/headset/headset_cent/empty(src)
-	new /obj/item/clothing/glasses/sunglasses(src)
-	new /obj/item/storage/backpack/satchel(src)
-	new /obj/item/pda/heads(src)
-	new /obj/item/clipboard(src)
-
-/obj/item/storage/box/syndie_kit/chameleon/broken/PopulateContents()
-	new /obj/item/clothing/under/chameleon/broken(src)
-	new /obj/item/clothing/suit/chameleon/broken(src)
-	new /obj/item/clothing/gloves/chameleon/broken(src)
-	new /obj/item/clothing/shoes/chameleon/noslip/broken(src)
-	new /obj/item/clothing/glasses/chameleon/broken(src)
-	new /obj/item/clothing/head/chameleon/broken(src)
-	new /obj/item/clothing/mask/chameleon/broken(src)
-	new /obj/item/storage/backpack/chameleon/broken(src)
-	new /obj/item/radio/headset/chameleon/broken(src)
-	new /obj/item/stamp/chameleon/broken(src)
-	new /obj/item/pda/chameleon/broken(src)
-	// No chameleon laser, they can't randomise for //REASONS//
-
-/obj/item/storage/box/syndie_kit/bee_grenades
-	name = "buzzkill grenade box"
-	desc = "A sleek, sturdy box with a buzzing noise coming from the inside. Uh oh."
-
-/obj/item/storage/box/syndie_kit/bee_grenades/PopulateContents()
-	for(var/i in 1 to 3)
-		new /obj/item/grenade/spawnergrenade/buzzkill(src)
+/obj/item/storage/box/syndicate

```

Comment:
i fucking had DM open (dmi editting i assume) while i was pulling remote and so now because carriage return line feed there are a few files with everything removed and added again, grumble grumble

Reply:
if there is any way to tell github to stop thinking i touched this file i'd be stoked to do that